playbook-website/app/javascript/components/MainSidebar/TopLevelNavItems.tsx

```diff
@@ -154,10 +154,10 @@ export const TopLevelNavItem = ({
           <>
             {name === "Components" ? (
               <>
-                {kits.map((link, i) => (
+                {kits.map((link, index) => (
                   <KitsNavItem
                     link={link}
-                    i={i}
+                    i={index}
```

Comment:
Same for prop names. Even though `i` is common in programming, it can still be confusing sometimes-- iterator, index, igloo (kidding). Let's be a little more verbose, for readability's sake. So if this `index` would clash with your param, you could try something like `itemIndex` or something.

Reply:
on it!